### PR TITLE
Rename ImportCompeted into ImportCompleted

### LINKED ISSUE #11960: `ICompositionImportedGpuImage.ImportCompeted` should be called `ImportCompleted`
**Describe the bug**
Here is the interface `ICompositionImportedGpuImage`:

https://github.com/AvaloniaUI/Avalonia/blob/81d455762d4b5dc49eab425898afd81ca131b99c/src/Avalonia.Base/Rendering/Composition/CompositionExternalMemory.cs#L87-L99

The member `ImportCompeted` should be called `ImportCompleted`.

**To Reproduce**
See the above source code.

**Expected behavior**
The member `ImportCompeted` should be called `ImportCompleted`.

**Screenshots**
N/A

**Desktop (please complete the following information):**

- OS: Linux (ArcoLinux), KDE/Wayland
- Version 11.0.0-rc1.1

### PR DESCRIPTION
**Pull request by jgcodes2020**
## What does the pull request do?

This PR properly fixes issue https://github.com/AvaloniaUI/Avalonia/issues/11960. 

## What is the current behavior?

The interface `Avalonia.Rendering.Composition.ICompositionImportedGpuObject` currently uses `ImportCompeted` for a `Task` that completes when the import is completed.

## What is the updated/expected behavior with this PR?

Practically identical, except that name is fixed.

## How was the solution implemented (if it's not obvious)?

`ImportCompeted` was renamed to `ImportCompleted`. A separate `ImportCompeted` property was added that simply fetches `ImportCompleted`. I have `[Obsolete]`d the `ImportCompeted` property to encourage people to switch over to the new spelling.

## Checklist

- [x] Added unit tests (if possible)? *(N/A)*
- [x] Added XML documentation to any related classes?
- [x] Consider submitting a PR to https://github.com/AvaloniaUI/Documentation with user documentation *(GPU interop docs are currently not available yet)*

## Breaking changes

See *Obsoletions/Deprecations* section below.

## Obsoletions / Deprecations

`ICompositionImportedGpuObject.ImportCompeted` is obsoleted (since it's a typo). The replacement is 
`ImportCompleted`.

I have yet to discuss with the core team. (@grokys @kekekeks @danwalmsley)

## Fixed issues

Fixes #11960

### CODE REVIEW

**Comment by jgcodes2020** [comment]
@dotnet-policy-service agree

File: src/Avalonia.Base/Rendering/Composition/CompositionExternalMemory.cs [hunk 1]
```diff
@@ -84,14 +84,21 @@ public enum CompositionGpuImportedImageSynchronizationCapabilities
...
     /// the user code is allowed to free the resource owner in case when a non-owning
-    /// sharing handle was used
+    /// sharing handle was used.
     /// </summary>
-    Task ImportCompeted { get; }
+    Task ImportCompleted { get; }
+    
+    /// <inheritdoc cref="ImportCompleted"/>
+    /// <seealso cref="ImportCompleted">ImportCompleted (recommended replacement)</seealso>
+    [Obsolete("Please use ICompositionGpuImportedObject.ImportCompleted instead")]
```
Comment (hez2010) [inline src/Avalonia.Base/Rendering/Composition/CompositionExternalMemory.cs:99]: Also hide it using [EditorBrowsable(EditorBrowsableState.Never)]?

Comment (jgcodes2020) [reply src/Avalonia.Base/Rendering/Composition/CompositionExternalMemory.cs:100]: just did that.

**Comment by maxkatz6** [comment]
@jgcodes2020 hi! We support netstandard2.0 and that's why you can't use default interface implementations. Instead, you need to implement this member independently.
Also couple of notes:
1. ICompositionGpuInterop should also be marked as not client implementabe
2. You will need to run "nuke ValidateApiDiff" from the repo root in the command line, so it will update breaking changes file.

**Review by maxkatz6 — APPROVED** [review]
Thanks!
Looks good to me. I fixed CI build as well.

**Comment by avaloniaui-team** [comment]
You can test this PR using the following package version. `11.0.999-cibuild0037700-beta`. (feed url: https://nuget-feed-all.avaloniaui.net/v3/index.json) [PRBUILDID]

**Pull request merged**